mne/time_frequency/csd.py

```diff
@@ -255,3 +221,187 @@ def compute_epochs_csd(epochs, mode='multitaper', fmin=0, fmax=np.inf,
                                              frequencies=frequencies[i],
                                              n_fft=n_fft))
         return csds
+
+
+@verbose
+def csd_array(X, sfreq, mode='multitaper', fmin=0, fmax=np.inf,
+              fsum=True, n_fft=None, mt_bandwidth=None,
+              mt_adaptive=False, mt_low_bias=True, verbose=None):
+    """Estimate cross-spectral density from an array.
+
+    .. note:: Results are scaled by sampling frequency for compatibility with
+              Matlab.
+
+    Parameters
+    ----------
+    X : array-like, shape (n_replicates, n_series, n_times)
```

Comment:
This is true in the most general sense, yes. But 90% of the time (or more?) it will be `(n_trials, n_series, n_times)` for MNE-Python'ers, so I'd prefer that. At the very least you could expand below to say "The time series data, e.g. with shape (n_trials, n_channels, n_times) or (n_trials, n_src, n_times)". But wait to see what @agramfort says.